root/static/scripts/common/i18n/expand2react.js

```diff
@@ -46,7 +46,7 @@ const condSubstThenTextContent = /^[^<>{}|]+/;
 const percentSign = /(%)/;
 const linkSubstStart = /^\{([0-9A-z_]+)\|/;
 const htmlTagStart = /^<(?=[a-z])/;
-const htmlTagName = /^(a|abbr|b|br|code|em|li|span|strong|ul)(?=[\s\/>])/;
+const htmlTagName = /^(a|abbr|b|br|code|em|li|p|span|strong|ul)(?=[\s\/>])/;
```

Comment:
Did you end up using `<p>` anywhere? Didn't see any places at a glance.

Reply:
The collection description is wrapped in <p> by default, apparently.

Reply:
Oh right, from format_wikitext